io.c

```diff
@@ -10584,14 +10583,18 @@ rb_ioctl(VALUE io, VALUE req, VALUE arg)
 
 /*
  *  call-seq:
- *     ios.ioctl(integer_cmd, arg)    -> integer
+ *    ioctl(integer_cmd, argument) -> integer
+ *
+ *  Calls system call {ioctl(2)}[https://linux.die.net/man/2/ioctl].
+ *
+ *  Issues a low-level command to an I/O device.
+ *
+ *  If argument +argument+ is an integer, it is passed directly;
+ *  if it is a string, it is interpreted as a binary sequence of bytes.
+ *
+ *  Arguments +integer_cmd+ and +argument+, as well as the returned value,
+ *  are platform-dependent.
```

Comment:
Again, I think this paragraph should immediately follow the "Issues a low-level command to an I/O device.", clearly signaling "Unless you know what you are doing, you don't need it. If you know, then here is argument types explanation:..."